mlrun/db/httpdb.py

```diff
@@ -525,10 +525,6 @@ def connect(self, secrets=None):
                 server_cfg.get("external_platform_tracking")
                 or config.external_platform_tracking
             )
-            config.model_endpoint_monitoring.store_type = (
```

Comment:
Does it have an implication for old clients?

Reply:
already in dev #6341